parm/gfs/monOznTime.yaml

```diff
@@ -0,0 +1,239 @@
+#
+# Time series plots for OznMon gfs
+#
+# Generate separate 30 day time/series plots for:
+#   - number of obs 
+#   - penalty
+#   - omgnbc
+#   - bias correction
+#   - omgbc
+
+diagnostics:
+
+# Data read
+# ---------
+datasets:
+  - name: time
+    satellite: npp
+    sensor: ompsnp
+    type: MonDataSpace
+    control_file:
+      - ../data/ozn_data/time/ompsnp_npp.ges.ctl
+    filenames:
+      - ../data/ozn_data/time/ompsnp_npp.ges.2023082118.ieee_d
```

Comment:
Just making a note here that one of the first things we should develop is the YAML generator using templating leveraging what we have in the global-workflow already

Reply:
Yes, that was one of the things I think we should discuss on Thurs.